httpsender/inject_js_in_html_page.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Inject Javascript code into a webpage.
+ *
+ *   - Code will be added inside <script></script> tags in the Response body's
+ *   <head></head> before sending it to the client.
+ *   - Code to be added is read from the file which path is `FILE` (this need to
+ *   be an absolute path, /tmp/test.js in our case).
+ */
+
+FILE = '/tmp/test.js'
+SCRIPT = '\t<script>' + loadScriptFromFile(FILE) + '</script>\n';
```

Comment:
Might worth doing this later, only when needed.

Reply:
Done.